v03_pipeline/lib/misc/io_test.py

```diff
@@ -45,15 +45,15 @@ def test_import_imputed_sex(self) -> None:
             [
                 hl.Struct(s='abc_1', predicted_sex='M'),
                 hl.Struct(s='abc_2', predicted_sex='F'),
-                hl.Struct(s='abc_3', predicted_sex='M'),
+                hl.Struct(s='abc_3', predicted_sex='U'),
             ],
         )
 
     def test_import_imputed_sex_unexpected_value(self) -> None:
         ht = import_imputed_sex(TEST_IMPUTED_SEX_UNEXPECTED_VALUE)
         self.assertRaisesRegex(
             hl.utils.java.HailUserError,
-            'Found unexpected value Unknown in imputed sex file',
+            'Found unexpected value UNKNOWN in imputed sex file',
```

Comment:
I'm turned around on this - I thought UNKNOWN is expected now ?

Reply:
This one is testing an "unexpected" value.  `Unknown` is now expected, but `UNKNOWN` is not.  `Unknown` was literally the value we chose to test as being unexpected previously just by happenstance 🤣

Reply:
Just to make it clear that `Unknown` is expected and `UNKNOWN` will still throw an error.  We're going to continue to be non-lenient here.

Reply:
Hmm I was about to suggest something a bit clearer, but this probably is good for being explicit about leniency